cmd_rate: 20
encoder_rate: 30

max_lin_speed: 2.0
min_lin_speed: -2.0
lin_speed_step: 2.0

max_ang_speed: 2.5
ang_steps: 12
step_len: 1.0

dead_man_button: True
dead_man_index: 5
dead_man_threshold: 0.7

ramp_trigger_button: False
ramp_trigger_index: 7
calib_trigger_button: False
calib_trigger_index: 7

steady_state_window: 1.5
steady_state_std_dev_threshold: 0.5

